components/slider/index.css

```diff
@@ -89,40 +120,39 @@ governing permissions and limitations under the License.
   z-index: auto;
 
   /* These calculations prevent the track from spilling outside of the control */
-  inline-size: calc(100% - calc(var(--spectrum-slider-controls-margin) * 2));
-  margin-inline-start: var(--spectrum-slider-controls-margin);
-  min-block-size: var(--spectrum-slider-height);
+  inline-size: calc(100% - calc(var(--mod-slider-controls-margin, var(--spectrum-slider-controls-margin))) * 2);
+  margin-inline-start: var(--mod-slider-controls-margin, var(--spectrum-slider-controls-margin));
+  min-block-size: var(--mod-slider-height-size, var(--spectrum-slider-height-size));
 
+  margin-block-end: var(--mod-slider-bottom-to-handle, var(--spectrum-slider-bottom-to-handle));
   vertical-align: top;
 }
 
 .spectrum-Slider-track,
 .spectrum-Slider-fill {
-  block-size: var(--spectrum-slider-track-height);
+  block-size: var(--mod-slider-track-thickness, var(--spectrum-slider-track-thickness));
```

Comment:
I think this needs to be declared at the top of the file like

```
.spectrum-Slider {
  --spectrum-slider-track-fill-thickness: var(--spectrum-slider-track-thickness);
}
```

And then used like

```
.spectrum-Slider-track,
.spectrum-Slider-fill {
  block-size: var(--mod-slider-track-fill-thickness, var(--spectrum-slider-track-fill-thickness));
}
```

Reply:
not sure actually. the var `mod-slider-track-thicknes` is coming directly from the core-tokens vars. 

``` 
"slider-track-thickness": {
    "sets": {
      "spectrum": {
        "value": "2px"
      },
      "express": {
        "value": "4px"
      }
    }
  },
````

@pfulton  is it ok to reference vars directly inside the Component from the variables.json file and adding the mods or do we need to assign it to another CSS variable first? I am fine either way, it's an easy fix.

Reply:
moved to top.